coderd/database/migrations/000071_provisioner_log_lines.up.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE provisioner_job_logs DROP COLUMN id;
+
+ALTER TABLE provisioner_job_logs ADD COLUMN id BIGSERIAL PRIMARY KEY; 
```

Comment:
Can we guarantee that adding this bigserial column will keep the same order of logs?

Reply:
It likely won't in all cases, but it seems to in most.

Having a _safe_ migration seems unreasonable here, so we settle for pretty great.